lua/mini/misc.lua

```diff
@@ -409,6 +413,49 @@ MiniMisc.zoom = function(buf_id, config)
   end
 end
 
+---
+MiniMisc.setup_goto_last_cursor = function()
+  if vim.fn.has("nvim-0.7.0") then
```

Comment:
Let's use only one version, which supports Neovim<0.7. No need to do same thing twice.

When deprecating Neovim<0.7, I'll replace all `vim.cmd()` calls with Lua autocommand API.

Reply:
Done.